packages/@atjson/offset-annotations/src/utils/video-urls.ts

```diff
@@ -176,21 +176,40 @@ function isBrightcoveURL(url: IUrl) {
   );
 }
 
+export enum Provider {
+  YOUTUBE,
```

Comment:
This will be serialized to JSON at some point, I've heard from folks at the bustle media group that having constants in their data makes understanding what's going on quite difficult. Could you make these string constants or a string union?

Reply:
@tim-evans for sure! Will update